src/mgmt/cli_mgmt.c

```diff
@@ -417,6 +444,78 @@ get_attach_info(const char *name, int *npsrs, struct dc_mgmt_psr **psrs,
 	return rc;
 }
 
+/*
+ * Get the CaRT network configuration for this client node
+ * via the get_attach_info() dRPC.
+ * Configure the client's local environment with these parameters
+ */
+int dc_mgmt_net_cfg(const char *name)
+{
+	int rc;
+	int npsrs;
+	char buf[SYS_INFO_BUF_SIZE];
+	char *ofi_interface;
+	char *ofi_domain;
+	struct sys_info sy_info;
+	struct dc_mgmt_psr *psrs;
+
+	if (name == NULL)
+		name = DAOS_DEFAULT_SYS_NAME;
+
+	/* Query the agent for the CaRT network configuration parameters */
+	rc = get_attach_info(name, &npsrs, &psrs, &sy_info);
```

Comment:
This returns -DER, but the rest of the calls in this routine, error codes you return are errno.

Reply:
ah i think you could use daos_errno2der() for the errno codes instead.

Reply:
Done.